src/fondant/runner.py

```diff
@@ -85,3 +85,36 @@ def get_name_from_spec(self, input_spec: str):
         with open(input_spec) as f:
             spec = yaml.safe_load(f)
             return spec["pipelineInfo"]["name"]
+
+
+class VertexRunner(Runner):
+    def __resolve_imports(self):
+        import google.cloud.aiplatform as aip
+
+        self.aip = aip
+
+    def __init__(self, project_id: str, project_region: str, service_account: str = ""):
+        self.__resolve_imports()
+
+        self.aip.init(
+            project=project_id,
+            location=project_region,
+        )
+        self.service_account = service_account
+
+    def run(self, input_spec: str, *args, **kwargs):
+        job = self.aip.PipelineJob(
+            display_name=self.get_name_from_spec(input_spec),
+            template_path=input_spec,
+            enable_caching=False,
+        )
+        if self.service_account:
+            job.submit(service_account=self.service_account)
+        else:
+            job.submit()
```

Comment:
```suggestion
         job.submit(service_account=self.service_account)
```

with the above suggestion, since the service account defined in `submit()` can be optional

Reply:
updated